app/assets/javascripts/application.js

```diff
@@ -13,5 +13,34 @@
 //= require rails-ujs
 //= require turbolinks
 //= require jquery
-//= require bootstrap.bundle.min.js
 //= require_tree .
+
+document.addEventListener('DOMContentLoaded', function () {
+
+  // Get all "navbar-burger" elements
+  var $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
+
+  // Check if there are any navbar burgers
+  if ($navbarBurgers.length > 0) {
+
+    // Add a click event on each of them
+    $navbarBurgers.forEach(function ($el) {
+      $el.addEventListener('click', function () {
+
+        // Get the target from the "data-target" attribute
+        var target = $el.dataset.target;
+        var $target = document.getElementById(target);
+
+        // Toggle the class on both the "navbar-burger" and the "navbar-menu"
+        $el.classList.toggle('is-active');
+        $target.classList.toggle('is-active');
+
+      });
+    });
+  }
+});
+
+$(document).on('click', '.notification > button.delete', function () {
```

Comment:
Also, I think we can use Native JS event delegation here instead of using jQuery. I'm starting to not be a fan of jQuery anymore mostly because the functionality offered by jQuery is in Javascript now.